src/nodejs/tsconfig.json

```diff
@@ -16,9 +16,11 @@
     "./../dotnet/Audio.UI.Blazor/Components/AudioRecorder/workers/audio-vad-worker.ts",
     "./../dotnet/Audio.UI.Blazor/Components/AudioRecorder/workers/opus-encoder-worker.ts",
     "./../dotnet/Audio.UI.Blazor/Components/AudioRecorder/workers/opus-encoder.ts",
-    "./../dotnet/UI.Blazor/ServiceWorkers/messaging-service-worker.ts",
     "./../dotnet/UI.Blazor/Layouts/base-layout.ts",
   ],
+  "exclude": [
+    "./../dotnet/UI.Blazor/ServiceWorkers/messaging-service-worker.ts",
```

Comment:
why excluded?

Reply:
We have another tsconfig for the service worker